tests/FacebookTest.php

```diff
@@ -68,55 +68,26 @@ public function testInstantiatingWithoutAppSecretThrows()
     }
 
     /**
-     * @expectedException \InvalidArgumentException
+     * @expectedException \Throwable
```

Comment:
must be reverted as `\Throwable` is only available since PHP 7, but we support 5.6